phpunit/functional/TicketTest.php

```diff
@@ -7046,6 +7081,17 @@ protected function timelineItemsProvider(): iterable
             ]
         );
 
+        $this->createItem(
+            \TicketTask::class,
+            [
+                'tickets_id'        => $ticket->getID(),
+                'content'           => 'private task assigned to see group',
+                'is_private'        => 1,
+                'groups_id_tech'    => $seegroup_id,
+                'date_creation'     => date('Y-m-d H:i:s', strtotime('+30s', $now)), // to ensure result order is correct
```

Comment:
In the previous `createItem()`, you would have to set `+40s` to avoid having the same time for each task.
```suggestion
                'date_creation'     => date('Y-m-d H:i:s', strtotime('+50s', $now)), // to ensure result order is correct
```